linera-core/src/client/mod.rs

```diff
@@ -1997,13 +1997,8 @@ where
 
         let incoming_bundles = self.pending_message_bundles().await?;
         let identity = self.identity().await?;
-        let info = self.chain_info_with_manager_values().await?;
-        let round = match Self::round_for_new_proposal(&info, &identity, None)? {
```

Comment:
`round_for_new_proposal` also doesn't need to take the block as an `Option` anymore